scripts/kodiak/pmm_avellaneda.py

```
@@ -0,0 +1,445 @@
+        - s = mid_price
+        - q = inventory (signed, positive for long, negative for short)
+        - γ = risk_aversion_gamma
+        - σ = volatility_sigma (annualized)
+        - τ = risk_horizon_tau_hours / 8760 (convert hours to years)
+        """
+        # Convert tau from hours to years
+        tau_years = self.config.risk_horizon_tau_hours / Decimal("8760")
+        
+        # Calculate inventory adjustment: q·γ·σ²·τ
+        inventory_adjustment = inventory * self.config.risk_aversion_gamma * \
+                               (self.config.volatility_sigma ** Decimal("2")) * tau_years
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)